### ISSUE TITLE
Calling openpgp.key.readArmored(pubkey) returns err: TypeError: (var).countUse is not a function

### ISSUE THREAD

**Issue by jano-m** [openpgpjs/openpgpjs#822]
```
const pubkey = `-----BEGIN PGP PUBLIC KEY BLOCK-----...`;
var publicKeys;
const readKey = async () => {
  let armored = await openpgp.key.readArmored(pubkey);
  publicKeys = armored.keys;
  console.log(armored);
};
readKey();

```
This will show in console the following:
```
{ keys: [],
  err:
   [ TypeError: (var).countUse is not a function
         at Object.transformPair (C:\Users\Jan\Documents\editor\node_modules\openpgp\dist\openpgp.js:23051:20)
         at Promise (C:\Users\Jan\Documents\editor\node_modules\openpgp\dist\openpgp.js:29722:65)
         at new Promise (<anonymous>)
         at Object.dearmor [as decode] (C:\Users\Jan\Documents\editor\node_modules\openpgp\dist\openpgp.js:29710:10)
         at Object.readArmored (C:\Users\Jan\Documents\editor\node_modules\openpgp\dist\openpgp.js:32247:41)
         at readKey (C:\Users\Jan\Documents\editor\config\pgp.js:48:35)
         at Object.<anonymous> (C:\Users\Jan\Documents\editor\config\pgp.js:52:1)
         at Object.<anonymous> (C:\Users\Jan\Documents\editor\config\pgp.js:79:3)
         at Module._compile (internal/modules/cjs/loader.js:693:30)
         at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10) ] } '    at readKey (C:\\Users\\Jan\\Documents\\editor\\config\\pgp.js:50:11)'
```

- Remark: I should mention, I am using Electron 4.0.0. It seems there is no problem with Electron 3.1.0 or 3.0.13, so this could be an electron issue?

**Comment by jano-m** [comment]
Thanks for the explanation. Yes, even TransformStream() throws the same exception.
However, setting TransformStream to undefined before calling require('openpgp') does not seem to help.

**Comment by twiss** [comment]
Could you verify that `typeof TransformStream === 'undefined'` afterwards? Perhaps try `delete global.TransformStream` or something like that.

Also, is the stack trace the same? There might be other internal functions throwing the same error.

**Comment by jano-m** [comment]
Quoting response by twiss from electron thread on this:

> This error comes from here: https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/streams/TransformStream.js?q=%5C.countUse&sq=package:chromium&l=69&dr=C&rcl=5a1cef8a88bd2c27675f736c1a403ddda6c5c72d. Even just new TransformStream() triggers it.

**Comment by twiss** [comment]
Alrighty. That error *is* caused by https://github.com/electron/electron/issues/16195#issuecomment-450990183. We try to create a `aes-{128,192,256}-cfb` cipher here. (Usually aes-256 but any of the three are possible depending on the message.)

**Comment by jano-m** [comment]
Thanks for explanation. Is there any other solution? We could use a different cipher, if that would work with native encryption and without changes to cfb.js.

**Comment by twiss** [comment]
Sure, search the README for "AEAD" for instructions for various other ciphers. But the changes above should happen anyway, it's a bug.

**Comment by twiss** [comment]
(Fixed in Electron 4.0.1)
